omod/src/main/compass/sass/providermanagement/bootstrap.scss

```diff
@@ -0,0 +1,6760 @@
+/*!
```

Comment:
The only other 2 places I could find bootstrap.scss was in pihcore and edtriage modules, which we probably do not want to include in here. I could add bootstrap.scss in the uicommons if this makes more sense?